modules/nodes-edge-remote-workers-strategies.adoc

```diff
@@ -0,0 +1,157 @@
+--
+
+[id="nodes-edge-remote-workers-strategies-tolerations_{context}"]
+Tolerations::
+You can use pod tolerations to mitigate the effects if the on-premise node controller adds a `node.kubernetes.io/unreachable` taint with a `NoExecute` effect to a node it cannot reach. 
+
+A taint with the `NoExecute` effect affects pods that are running on the node as follows:
+
+* Pods that do not tolerate the taint are queued for eviction.
+* Pods that tolerate the taint without specifying a `tolerationSeconds` value in their toleration specification remain bound forever.
+* Pods that tolerate the taint with a specified `tolerationSeconds` value remain bound for the specified amount of time.  Once the time is expired, the pods are queued for eviction.
+
+You can delay or avoid pod eviction by configuring pods tolerations with the `NoExecute` effect for the `node.kubernetes.io/unreachable` and `node.kubernetes.io/not-ready` taints.
+
+.Example toleration in a pod spec
+[source,yaml]
+----
+...
+tolerations:
+- key: "node.kubernetes.io/unreachable"
+  operator: "Exists"
+  effect: "NoExecute" <1>
+  tolerationSeconds: 0
+- key: "node.kubernetes.io/not-ready"
+  operator: "Exists"
+  effect: "NoExecute" <2>
+  tolerationSeconds: 0
+...
+----
+<1> The `NoSchedule` effect with `tolerationSeconds`: 0 allows pods to remain if the control plane cannot reach the node.
+<2> The `NoSchedule` effect with `tolerationSeconds`: 0 allows pods to remain if the control plane marks the node as `Unhealthy`.
+
+{product-title} uses the `tolerationSeconds` value after the `pod-eviction-timeout` value has elapsed.  
+
+Other types of {product-title} objects::
+You can use ReplicaSets, Deployments, and ReplicationControllers. The scheduler can reschedule these pods onto other nodes after the node is disconnected for 5 minutes. Rescheduling onto other nodes can be beneficial for some workloads, such as REST APIs, where an administrator can guarantee a specific number of pods are running and accessible.
+
+[NOTE]
+====
+When working with remote worker nodes, rescheduling pods on different nodes might not be desirable if remote worker nodes are intended to be reserved for specific functions.
+====
+
+[id="nodes-edge-remote-workers-strategies-statefulset_{context}"]
+https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/[StatefulSets] do not get restarted when there is an outage. The pods remain in the `terminating` state until the control plane can acknowledge the pods are terminated. 
+
+{product-title} cannot migrate pods that require persistent volumes to other zones in the case of network separation to avoid scheduling a to a node that does not have access to the same type of persistent storage.
```

Comment:
```suggestion
{product-title} cannot migrate pods that require persistent volumes to other zones in the case of network separation to avoid scheduling a to a node that does not have access to the same type of persistent storage.
```
```suggestion
To avoid scheduling a to a node that does not have access to the same type of persistent storage, {product-title} cannot migrate pods that require persistent volumes to other zones in the case of network separation.
```